spec/miq_ae_event_spec.rb

```diff
@@ -80,7 +101,7 @@
         it "has tenant" do
           args = {:user_id => admin.id, :miq_group_id => vm_group.id, :tenant_id => vm_group.tenant.id}
           expect(MiqAeEngine).to receive(:deliver_queue).with(hash_including(args), anything)
-
+          expect(MiqServer).to receive(:my_zone).and_return('default')
```

Comment:
@tinaafitz 
Can we have something like this
before { allow(MiqServer).to receive_messages(:my_zone => "default") }
then we dont have to change every line